app/views/shared/_footer.html.erb

```diff
@@ -4,10 +4,7 @@
 			<%= link_to image_tag("neos-logo.jpg"), "https://www.neoslibraries.ca" %>
 		</div>
 		<div class="col-sm-6 footer-links">
-			 <%- if cookies[:brand] %>
-       		 <%- $libraryname = @libraries[cookies[:brand]]["name"] %>
-       		 <%- $homeurl = @libraries[cookies[:brand]]["url"] %>
-       		 <%- $neosurl = @libraries[cookies[:brand]]["neosurl"] %>
+			 <%- if $brand %>
```

Comment:
I think this might also fix a problem with the email this form.

Reply:
really? how? Gah was my stuff breaking that?